arroyo-controller/src/lib.rs

```diff
@@ -459,45 +457,10 @@ impl ControllerGrpc for ControllerServer {
             }
         };
 
-        let mut function_name = None;
-        {
-            let result = match parse_file(definition.as_str()) {
-                Ok(result) => result,
-                Err(e) => {
-                    return Ok(udf_error_resp(e));
-                }
-            };
-
-            for item in result.items {
-                match item {
-                    Item::Fn(f) => {
-                        if function_name.is_some() {
-                            return Ok(Response::new(CheckUdfsResp {
-                                errors: vec!["Only one function is allowed in UDFs".to_string()],
-                                udf_name: None,
-                            }));
-                        }
-                        function_name = Some(f.sig.ident.to_string());
-                    }
-                    Item::Use(_) => {}
-                    _ => {
-                        return Ok(Response::new(CheckUdfsResp {
-                            errors: vec![
-                                "Only functions and use statements are allowed in UDFs".to_string()
-                            ],
-                            udf_name: None,
-                        }))
-                    }
```

Comment:
As far as I know we don't actually need this check any more, so I didn't implement it in the ArroyoSchemaProvider.